app/src/main/java/com/cmput301f20t21/bookfriends/ui/profile/ProfileEditDialog.java

```diff
@@ -35,8 +33,10 @@ public View onCreateView(final LayoutInflater inflater, @Nullable ViewGroup cont
         editPhone = view.findViewById(R.id.edit_phone);
         cancel = view.findViewById(R.id.text_cancel);
         confirm = view.findViewById(R.id.text_confirm);
-        existEmail = view.findViewById(R.id.email);
-        existPhone = view.findViewById(R.id.phone);
+        //set value from the fragment
+        editEmail.setText(getArguments().getString("email"));
```

Comment:
we should first get the arguments by doing `Bundle arguments = getArguments()` and check if it is null before proceeding to get the Strings

without doing this check there is a possibility that the arguments did not get passed in to the fragment and that will produce a NPE(Null Pointer Exception) and cause our app to crash